deepl-connector-demo/dataclasses/com/axonivy/connector/deepl/demo/Data.ivyClass

```diff
@@ -1,2 +1,8 @@
 Data #class
 com.axonivy.connector.deepl.demo #namespace
+inputText String #field
```

Comment:
having this data only on the dialog was the right place ... now we kind of mix it with the demo ... for no obvious reason